wiki/Client/Options/de.md

```diff
@@ -1,292 +1,431 @@
-| Genauigkeitsmeter (stellt die Erscheinung des Genauigkeitsmessers am unteren Bildschirmrand fest). | **Aus:** Zeigt keinen Genauigkeitsmeter, **Farbe:** Zeigt voherige Hitwerte an (basiert an den Farben vom Standardskin für die 300/100/50/Miss), **Abweichung \[Standard\]:** Zeigt Ihnen präzise, wie akkurat Sie bei jedem Hit sind. Die Länge der einzelnen Bereiche hängen von der OD ab. Links - Mitte - Rechts = zu früh - Perfekt – zu spät |
-| Genauigkeitsmetergröße | Passt die Größe der Genauigkeitsmeterangezeige an. |
-| Tasten-Overlay | Immer das Tasten-Overlay anzeigen, selbst wenn Sie lokal spielen. Nützlich zum Aufnehmen oder Streamen. |
-| Approach Circle des ersten Objektes beim Mod "Hidden" | Dies erlaubt Ihnen, den Approach Circle des ersten Hit-Objektes sehen zu können. |
-| osu!mania-Scrollgeschwindigkeit anhand BPM skalieren | Die Scrollgeschwindigkeit wird proportional zur Grund-BPM der Beatmap eingestellt. Wenn Sie dies aktivieren, scrollen schnellere Beatmaps schneller. |
-| osu!mania-Scrollgeschwindigkeit für jede Beatmap merken | Speichert die Scrollgeschwindigkeiten für jede Beatmap einzeln. |
+![Symbol für die Sektion Audio](img/audio.jpg "Symbol für die Sektion Audio")
 
-### Audio
+In diesem Abschnitt geht es um audiobezogene Optionen.
 
-#### Lautstärke
+### Geräte
 
-The volume controls allow one to change the volume in-game.
+| Name | Beschreibung | Typ | Standardwert |
+| :-- | :-- | :-- | :-- |
+| `Wiedergabegeräte` | Wähle dein bevorzugtes Wiedergabegerät für die Audio aus. (Die angegebenen Optionen hängen davon ab, was dein Computer unterstützt.) | Dropdown-Menü | `Standard` |
+| `Audio compatibility mode` | Benutzt die alte Audio-Engine, die eine höhere Latenzzeit hat, aber kompatibler ist. | Checkbox | `Deaktiviert` |
 
-- Allgemein: Allgemeiner Lautstärkeregler.
-- Musik: Beeinflusst nur die Musik selbst.
-- Effekte: Beeinflusst die Lautstärke der Hitsounds, sowie andere Soundeffekte.
+### Lautstärke
 
-| Name | Effekt |
-| :-- | :-- |
-| Beatmaps Hitsounds ignorieren | Hitsounds, die einer Beatmap beigefügt worden sind, werden ignoriert. |
+| Name | Beschreibung | Typ | Standardwert |
+| :-- | :-- | :-- | :-- |
+| `Allgemein` | Kontrolliert die allgemeine Lautstärke. | Slider | `100%` |
+| `Musik` | Kontrolliert die Lautstärke der Musik. | Slider | `80%` |
+| `Effekte` | Kontrolliert die Lautstärke der Hitsounds sowie anderer Soundeffekte im Spiel. | Slider | `80%` |
+| `Hitsounds ignorieren` | Bevorzugt Hitsounds, die vom aktuellen Skin bereitgestellt werden, anstatt die Hitsounds, die in der Beatmap enthalten sind. | Checkbox | `Deaktiviert` |
 
-#### Universeller Audio-Offset
+Die allgemeine, Musik- sowie Effektelautstärke kann an anderer Stelle auch durch das Drücken von `Alt` sowie Hoch- und Runterscrollen oder durch das Drücken von `Alt` sowie `Up` oder `Down` eingestellt werden.
```

Comment:
```suggestion
Die allgemeine, Musik- sowie Effektelautstärke kann an anderer Stelle auch durch das Drücken von `Alt` sowie Hoch- und Runterscrollen oder durch das Drücken von `Alt` sowie `nach oben` oder `nach unten` eingestellt werden.
```
should be the official german names i believe, otherwise using ``Pfeiltaste hoch` and `Pfeiltaste runter` could also work